src/ansys/tools/path/path.py

```diff
@@ -999,7 +999,7 @@ def _get_application_path(
         exe_loc = _prompt_path(product)
         _change_default_path(product, exe_loc)
         return exe_loc
-
+    warnings.warn(f"No path found for {product} in default locations")
```

Comment:
```suggestion
    warnings.warn(f"No path found for {product} in default locations.")
```
Warnings and messages should be complete sentences, including ending with punctuation.